QUEST_LV_0100.tsv

```diff
@@ -4810,66 +4810,66 @@ QUEST_LV_0100_20150317_004809	Collect demon souls
+QUEST_LV_0100_20150317_004837	$Recharge the Strong Magical Power Absorption Jar
+QUEST_LV_0100_20150317_004838	$Coben showed you a device his brother left him. Could this device help one gain access to the Mage Tower? Charge the device first.
+QUEST_LV_0100_20150317_004839	$The device is now fully charged. Talk to Coben.
+QUEST_LV_0100_20150317_004840	$You may meet Coben's brother soon. Talk to Coben.
+QUEST_LV_0100_20150317_004841	$Check the writings on the Commanding Monument
 QUEST_LV_0100_20150317_004842	The name of the epitaph is strange which should be the last epitaph. Check what is written on it.
 QUEST_LV_0100_20150317_004843	Unfortunately, it seems that Coben's brother was dead after failing going up the Mage Tower. Tell Coben about it.
-QUEST_LV_0100_20150317_004844	demon glove treatment
-QUEST_LV_0100_20150317_004845	Talk to Karoline Canolin
-QUEST_LV_0100_20150317_004846	You met Karoline Canolin who seems very anxious. Talk to Karolilne Canolin.
-QUEST_LV_0100_20150317_004847	Defeat Cockatrice
-QUEST_LV_0100_20150317_004848	{memo X}Karoline Canolin is worried about the cockarties that is blocking the way to Fedimian. Defeat Cockatries that is blocking the way to Fedimian.
-QUEST_LV_0100_20150317_004849	Report to Karoline Canolin.
-QUEST_LV_0100_20150317_004850	You defeated enough Cockatries. Report to Karoline Canolin.
-QUEST_LV_0100_20150317_004851	Karoline Canolin told you that the monsters stole all the tools. Talk to Karoline Canolin again.
+QUEST_LV_0100_20150317_004844	$Defeat the Cursed Devilglove
+QUEST_LV_0100_20150317_004845	$Talk to Karoline Canolyn
+QUEST_LV_0100_20150317_004846	$You met Karoline Canolyn who seems very anxious. Talk to Karoline Canolyn.
+QUEST_LV_0100_20150317_004847	$Defeat Cockatrices
+QUEST_LV_0100_20150317_004848	{memo X}Karoline Canolyn is worried about the Cockatrices that is blocking the way to Fedimian. Defeat Cockatrices that is blocking the way to Fedimian.
+QUEST_LV_0100_20150317_004849	$Report to Karoline Canolyn.
+QUEST_LV_0100_20150317_004850	$You defeated enough Cockatrices. Report to Karoline Canolyn.
+QUEST_LV_0100_20150317_004851	$Karoline Canolyn told you that the monsters stole all the tools. Talk to Karoline Canolyn again.
 QUEST_LV_0100_20150317_004852	Defeat Moa
-QUEST_LV_0100_20150317_004853	{memo X}Karoline Canolin told you that a monster called Moa stole the tool box. Defeat the moa that ran away to the upper side of the abondoned farm and retrieve the tool box.
-QUEST_LV_0100_20150317_004854	Unfortunately, the tool box wasn't there. Go back and talk to Karoline Canolin.
+QUEST_LV_0100_20150317_004853	{memo X}Karoline Canolyn told you that a monster called Moa stole the tool box. Defeat the Moa that ran away to the upper side of the Abandoned Farm and retrieve the tool box.
+QUEST_LV_0100_20150317_004854	$Unfortunately, the toolbox wasn't there. Go back and talk to Karoline Canolyn.
```

Comment:
Go back -> Return 

(I've replaced a lot of "Go Back"s to "Return"s in my quest_lv_0100 PR.